packages/uikit/src/components/Select/Select.tsx

```diff
@@ -122,7 +114,8 @@ const Select: React.FunctionComponent<React.PropsWithChildren<SelectProps>> = ({
     event.stopPropagation();
   };
 
-  const onOptionClicked = (selectedIndex: number) => () => {
+  const onOptionClicked = (selectedIndex: number) => (e: React.MouseEvent) => {
+    e.stopPropagation();
```

Comment:
may I ask the reason for adding this?

Reply:
e.stopPropagation prevents the widget from closing if you click it. with out adding this the notification modal closes if you click the select dropdown